homeassistant/components/sensibo/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
+  # Bronze
+  config-flow:
+    status: todo
+    comment: |
+      Data descriptions.
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities doesn't subscribe to events.
+  dependency-transparency: done
+  action-setup:
```

Comment:
Why is assumed state a service?
```py
    platform.async_register_entity_service(
        SERVICE_ASSUME_STATE,
        {
            vol.Required(ATTR_STATE): vol.In(["on", "off"]),
        },
        "async_assume_state",
    )
```

Reply:
The user can use the service to set the device in sync with HA e.g. the device is in reality on, but Sensibo thinks it's off. The user can use this service to tell Sensibo api the device is actually on.

Reply:
How does sensibo not know if the device is on though? We set it on via their service right?

Reply:
The user could also use the physical remote to the device (think a heatpump on the wall for your house) and Sensibo doesn't pick up the ir signal (and records the device has been turned on or off).
Quite a common use-case that you need to sync up.